api/metadata_handler.go

```diff
@@ -45,19 +51,29 @@ func apiMetadataTriageHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	githubBotClient, err := shared.GetGithubClientFromToken(ctx, "github-wpt-fyi-bot-token")
+	wptfyiBotClient, err := shared.GetGithubClientFromToken(ctx, "github-wpt-fyi-bot-token")
 	if err != nil {
-		http.Error(w, "Unable to get Github Client: "+err.Error(), http.StatusInternalServerError)
+		http.Error(w, "Unable to get wptfyiBotClient: "+err.Error(), http.StatusInternalServerError)
 		return
 	}
 
 	aeAPI := shared.NewAppEngineAPI(ctx)
-	git := shared.GetMetadataGithub(githubBotClient, user.GitHubHandle, user.GithuhEmail)
+	git := shared.GetMetadataGithub(wptfyiBotClient, user.GitHubHandle, user.GithuhEmail)
 	log := shared.GetLogger(ctx)
-	fetcher := webappMetadataFetcher{ctx: ctx, client: aeAPI.GetHTTPClient(), url: shared.MetadataArchiveURL}
+	gitHubClient, err := shared.NewAppEngineAPI(ctx).GetGitHubClient()
+	if err != nil {
+		http.Error(w, "Unable to get Github client: "+err.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	fetcher := webappMetadataFetcher{
+		ctx:          ctx,
+		httpClient:   aeAPI.GetHTTPClient(),
+		gitHubClient: gitHubClient,
```

Comment:
Why don't we just use `wptfyiBotClient` here? It has broader scopes but the fetcher is read-only anyway.

Reply:
Yea that would work here. I was worried that at peak wptfyiBotClient is more likely to reach the GitHub request limit